uniffi_macros/src/export/attributes.rs

```diff
@@ -224,25 +236,37 @@ impl ToTokens for AsyncRuntime {
     }
 }
 
-#[derive(Default)]
-pub struct NonExportArgs {
+/// Arguments for function inside an impl block
+///
+/// This stores the parsed arguments for `uniffi::constructor` and `uniffi::method`
+#[derive(Clone, Default)]
+pub struct ExportedImplFnArgs {
     pub(crate) name: Option<String>,
+    pub(crate) defaults: DefaultMap,
 }
```

Comment:
This is almost the same as `ExportFnArgs`, but it doesn't store the async runtime.  I think we can combine these into 1 struct to simplify the code.  Allowing users to specify an async runtime for a single method seems like a nice side benefit as well.  However, let's do that in follow-up PR.